docs/source/documentation.rst

```diff
@@ -495,19 +490,8 @@ Here is a list of useful Jupyter notebooks available with AxonDeepSeg:
 * `02-morphometrics_extraction.ipynb <https://github.com/neuropoly/axondeepseg/blob/master/notebooks/04-morphometrics_extraction.ipynb>`_:
     Notebook that shows how to extract morphometrics from a sample segmented with AxonDeepSeg. The user can extract and save morphometrics for each axon (diameter, solidity, ellipticity, centroid, ...), estimate aggregate morphometrics of the sample from the axon/myelin segmentation (g-ratio, AVF, MVF, myelin thickness, axon density, ...), and generate overlays of axon/myelin segmentation masks, colocoded for axon diameter.
 
-.. NOTE ::
-    If it is the first time, install the Jupyter notebook package in the terminal::
-
-        pip install jupyter
-
-    Then, go to the notebooks/ subfolder of AxonDeepSeg and launch a particular notebook as follows::
-
-        cd notebooks
-        jupyter notebook name_of_the_notebook.ipynb 
```

Comment:
We could keep this part to launch the notebook and only delete the "pip install" part.

Reply:
Done in 55795b02